src/master-playlist-controller.js

```diff
@@ -1431,7 +1431,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
   onSyncInfoUpdate_() {
     let audioSeekable;
 
-    if (!this.masterPlaylistLoader_) {
+    if (!this.masterPlaylistLoader_ || this.sourceUpdater_.hasCreatedSourceBuffers()) {
```

Comment:
This was causing us to:
1. As soon as audio/video segment loader fires this we come in here
2. Since only one of the loaders is actually ready at this point, we get seekable from just that one
3. We seek to the end of this invalid seekable range
4. We come back in here again and this time we get an intersection between the two and a valid seekable
5. We seek to the valid seekable end via playback watcher
6. This causes content to be removes and things to be downloaded/appended again.